src/NodaTime/Period.cs

```diff
@@ -423,6 +423,20 @@ public static int DaysBetween(LocalDate start, LocalDate end)
             return InternalDaysBetween(start, end);
         }
 
+        /// <summary>
+        /// Returns the difference between start and end time in nanoseconds.
```

Comment:
Perhaps `Returns the number of nanoseconds between two <see cref="LocalTime"/> objects.` ? (following `DaysBetween()` above)

Reply:
Actually I have been thinking if there could be better description as well. 

A few quote from other methods:
1. Between(LocalTime start, LocalTime end)
`Returns the exact difference between two times.`

2. Between(LocalTime start, LocalTime end, PeriodUnits units)
`Returns the period between a start and an end time, using only the given units.`

3. Between(LocalDate start, LocalDate end)
`Returns the exact difference between two dates.`

4. DaysBetween(LocalDate start, LocalDate end)
`Returns the number of days between two <see cref="LocalDate"/> objects.`

Style of 4 looks absolutely fine to me, if you all agree I can submit a change for it.

Reply:
Yup, the style of 4 looks right to me.

Reply:
Updated both summary and return section to align with the DaysBetween()'s description